src/lib/components/modal/modal.js

```diff
@@ -0,0 +1,67 @@
+import React, { useEffect } from "react";
+import PropTypes, { element } from "prop-types";
+import styles from "./modal.module.scss";
+
+const Modal = props => {
+  const {
+    children,
+    showModal,
+    handleModal,
+    backgroundColor,
+    backGroundStyle,
+    hideBodyOverflowY,
+  } = props;
+
+  useEffect(() => {
+    // This is manipulating the parent DOM of the react virtual DOM, so I think its going to be OK for what we need?
+    // we would have to hide overflow for the body and app and then make them fill the page and then adjust the react elements to be able to controll overflow-y in css, I think.
+    document.body.style.overflowY =
```

Comment:
I think we can run a check for document, and it is also only available when the vidoe is ready, which should be client side right?
Also it could be done with css and some extra divs but I think thats probably too complex just for hiding body scroll when modal is active.

I am happy to do anything with that we dont need to hide scroll, I'm open to any suggestion. 👍